ets-demo/etstool.py

```diff
@@ -311,6 +336,62 @@ def flake8(runtime, toolkit, environment):
     execute(commands, parameters)
 
 
+@cli.command(name="generate-bundles")
+def generate_bundles():
+    """ Generate application bundles for each platform to be uploaded to
+    enthought/free repository
+    """
+
+    for platform in PLATFORMS:
+        bundle_dir = os.path.join("bundle", platform)
+        if not os.path.exists(bundle_dir):
+            os.makedirs(bundle_dir)
+
+        bundle_file = os.path.join(
+            bundle_dir,
+            f"etsdemo_{platform.replace('-', '_')}-{APP_BUNDLE_VERSION}.bundle"
```

Comment:
per the hatcher docs: "The filename of the bundle is expected to follow the <name-with-underscores>-<version>-<build>.bundle template." 

However, IIUC we will need to upload a bundle for every platform we wish to support? 
"Exported bundles are platform-specific, and cannot be used to reproduce a similar environment on a different platform, e.g. you cannot use an osx-x86_64 bundle to create an environment on a Linux machine."

Reply:
> However, IIUC we will need to upload a bundle for every platform we wish to support?

That is correct but they don't need to have unique names. When the upload happens, the server parses the incoming file and understands what platform the bundle is used for.

So, we can remove the platform names from the bundle files.